### YamlMime:ManagedReference
items:
  - uid: com.microsoft.azure.management.cdn._error_response_exception
    id: _error_response_exception
    parent: com.microsoft.azure.management.cdn
    children:
      - com.microsoft.azure.management.cdn._error_response_exception.body()
      - 'com.microsoft.azure.management.cdn._error_response_exception.ErrorResponseException(final String,final Response<ResponseBody>)'
      - 'com.microsoft.azure.management.cdn._error_response_exception.ErrorResponseException(final String,final Response<ResponseBody>,final ErrorResponse)'
    href: com.microsoft.azure.management.cdn._error_response_exception.yml
    langs:
      - java
    name: ErrorResponseException
    nameWithType: ErrorResponseException
    fullName: com.microsoft.azure.management.cdn.ErrorResponseException
    type: Class
    source:
      remote: &ref_0
        path: azure-mgmt-cdn/src/main/java/com/microsoft/azure/management/cdn/ErrorResponseException.java
        branch: v1.3.0
        repo: 'https://github.com/Azure/azure-sdk-for-java'
      path: azure-mgmt-cdn/src/main/java/com/microsoft/azure/management/cdn/ErrorResponseException.java
      startLine: 17
    package: com.microsoft.azure.management.cdn
    summary: <p>無効な応答の例外のスロー <xref uid="com.microsoft.azure.management.cdn._error_response" data-throw-if-not-resolved="false">ErrorResponse</xref>情報。 </p>
    syntax: &ref_1
      content: public class ErrorResponseException
    inheritance:
      - java.lang.Object
      - RestException
  - uid: com.microsoft.azure.management.cdn._error_response_exception.body()
    id: body()
    parent: com.microsoft.azure.management.cdn._error_response_exception
    href: com.microsoft.azure.management.cdn._error_response_exception.yml
    langs:
      - java
    name: body()
    nameWithType: ErrorResponseException.body()
    fullName: ErrorResponse com.microsoft.azure.management.cdn.ErrorResponseException.body()
    overload: com.microsoft.azure.management.cdn._error_response_exception.body*
    type: Method
    source:
      remote: *ref_0
      path: azure-mgmt-cdn/src/main/java/com/microsoft/azure/management/cdn/ErrorResponseException.java
      startLine: 40
    package: com.microsoft.azure.management.cdn
    syntax:
      content: public ErrorResponse body()
      return:
        type: com.microsoft.azure.management.cdn._error_response
  - uid: 'com.microsoft.azure.management.cdn._error_response_exception.ErrorResponseException(final String,final Response<ResponseBody>)'
    id: 'ErrorResponseException(final String,final Response<ResponseBody>)'
    parent: com.microsoft.azure.management.cdn._error_response_exception
    href: com.microsoft.azure.management.cdn._error_response_exception.yml
    langs:
      - java
    name: 'ErrorResponseException(final String message, final Response<ResponseBody> response)'
    nameWithType: 'ErrorResponseException.ErrorResponseException(final String message, final Response<ResponseBody> response)'
    fullName: 'com.microsoft.azure.management.cdn.ErrorResponseException.ErrorResponseException(final String message, final Response<ResponseBody> response)'
    overload: com.microsoft.azure.management.cdn._error_response_exception.ErrorResponseException*
    type: Constructor
    source:
      remote: *ref_0
      path: azure-mgmt-cdn/src/main/java/com/microsoft/azure/management/cdn/ErrorResponseException.java
      startLine: 24
    package: com.microsoft.azure.management.cdn
    summary: |-
      <p>新しいインスタンスを初期化、 <xref uid="com.microsoft.azure.management.cdn._error_response_exception" data-throw-if-not-resolved="false">ErrorResponseException</xref>クラスです。</p>
      <p></p>
    syntax:
      content: 'public ErrorResponseException(final String message, final Response<ResponseBody> response)'
      parameters:
        - id: message
          type: 3d93591e
          description: <p>例外メッセージや、メッセージが利用できない場合は、応答コンテンツ </p>
        - id: response
          type: 4f402e61
          description: <p>HTTP 応答 </p>
  - uid: 'com.microsoft.azure.management.cdn._error_response_exception.ErrorResponseException(final String,final Response<ResponseBody>,final ErrorResponse)'
    id: 'ErrorResponseException(final String,final Response<ResponseBody>,final ErrorResponse)'
    parent: com.microsoft.azure.management.cdn._error_response_exception
    href: com.microsoft.azure.management.cdn._error_response_exception.yml
    langs:
      - java
    name: 'ErrorResponseException(final String message, final Response<ResponseBody> response, final ErrorResponse body)'
    nameWithType: 'ErrorResponseException.ErrorResponseException(final String message, final Response<ResponseBody> response, final ErrorResponse body)'
    fullName: 'com.microsoft.azure.management.cdn.ErrorResponseException.ErrorResponseException(final String message, final Response<ResponseBody> response, final ErrorResponse body)'
    overload: com.microsoft.azure.management.cdn._error_response_exception.ErrorResponseException*
    type: Constructor
    source:
      remote: *ref_0
      path: azure-mgmt-cdn/src/main/java/com/microsoft/azure/management/cdn/ErrorResponseException.java
      startLine: 35
    package: com.microsoft.azure.management.cdn
    summary: |-
      <p>新しいインスタンスを初期化、 <xref uid="com.microsoft.azure.management.cdn._error_response_exception" data-throw-if-not-resolved="false">ErrorResponseException</xref>クラスです。</p>
      <p></p>
    syntax:
      content: 'public ErrorResponseException(final String message, final Response<ResponseBody> response, final ErrorResponse body)'
      parameters:
        - id: message
          type: 3d93591e
          description: <p>例外メッセージや、メッセージが利用できない場合は、応答コンテンツ </p>
        - id: response
          type: 4f402e61
          description: <p>HTTP 応答 </p>
        - id: body
          type: 93de0ef9com.microsoft.azure.management.cdn._error_response
          description: <p>逆シリアル化された応答本文 </p>
references:
  - uid: 3d93591e
    spec.java:
      - name: final String
        fullName: final String
  - uid: 4f402e61
    spec.java:
      - name: final Response<ResponseBody>
        fullName: final Response<ResponseBody>
  - uid: com.microsoft.azure.management.cdn._error_response_exception.ErrorResponseException*
    name: ErrorResponseException
    nameWithType: ErrorResponseException.ErrorResponseException
    fullName: com.microsoft.azure.management.cdn.ErrorResponseException.ErrorResponseException
    package: com.microsoft.azure.management.cdn
  - uid: 93de0ef9com.microsoft.azure.management.cdn._error_response
    spec.java:
      - name: 'final '
        fullName: 'final '
      - uid: com.microsoft.azure.management.cdn._error_response
        name: ErrorResponse
        fullName: com.microsoft.azure.management.cdn.ErrorResponse
        href: com.microsoft.azure.management.cdn._error_response.yml
  - uid: com.microsoft.azure.management.cdn._error_response_exception.body*
    name: body
    nameWithType: ErrorResponseException.body
    fullName: ErrorResponse com.microsoft.azure.management.cdn.ErrorResponseException.body
    package: com.microsoft.azure.management.cdn
  - uid: com.microsoft.azure.management.cdn._error_response_exception
    parent: com.microsoft.azure.management.cdn
    href: com.microsoft.azure.management.cdn._error_response_exception.yml
    name: ErrorResponseException
    nameWithType: ErrorResponseException
    fullName: com.microsoft.azure.management.cdn.ErrorResponseException
    type: Class
    summary: <p>無効な応答の例外のスロー <xref uid="com.microsoft.azure.management.cdn._error_response" data-throw-if-not-resolved="false">ErrorResponse</xref>情報。 </p>
    syntax: *ref_1
  - uid: com.microsoft.azure.management.cdn._error_response
    parent: com.microsoft.azure.management.cdn
    href: com.microsoft.azure.management.cdn._error_response.yml
    name: ErrorResponse
    nameWithType: ErrorResponse
    fullName: com.microsoft.azure.management.cdn.ErrorResponse
    type: Class
    summary: <p>エラー応答は、CDN サービスが受信要求を処理することを示します。 その理由は、エラー メッセージで提供されます。 </p>
    syntax:
      content: public class ErrorResponse